templates/extension-template-js/index.js

```diff
@@ -26,12 +28,24 @@ var _globalName_ = (function (jspsych) {
 
     on_finish(params) {
       return {
-        data_property: "data_value",
+        data_name: 99, // Make sure this type and name matches data_name
```

Comment:
makes it sound like the parameter name has to be `data_name`

Reply:
Changed to data1 and data2, thoughts?

Reply:
Wasn't clear at all, sorry. I meant more that the comment should say that it needs to match whatever name is in the info object.

Reply:
Adjusted to be more specific referencing the naming and typing for the corresponding object within the data object with const<info>.